src/String(lesson5)/stringSearch.js

```diff
@@ -0,0 +1,54 @@
+const data = [
```

Comment:
You can try to rename this data, it's a really bad name, it can be everithing in variable with such a name. It says absolutely nothing about data inside
BTW You can create a folder for such mock data.